vision/automl/requirements.txt

```diff
@@ -1 +1 @@
-google-cloud-automl==0.2.0
```

Comment:
Fixed in [this PR](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2318)